VideoLocker/AndroidManifest.xml

```diff
@@ -217,6 +226,37 @@
         <service android:name="org.edx.mobile.services.DownloadSpeedService" />
         <service android:name="org.edx.mobile.services.FetchCourseFriendsService"/>
 
+
+        <!--   Parse Setup -->
+        <!--  https://www.parse.com/apps/quickstart#parse_push/android/existing  -->
+        <service android:name="com.parse.PushService" />
```

Comment:
Seems fine to wait for now. At some point we'll need to figure this out though. Hopefully we can leverage the "manifestMerge" gradle option.